concent_api/core/tests/utils.py

```diff
@@ -265,21 +284,21 @@ def _get_deserialized_ack_report_computed_task(
 
     def _get_serialized_ack_report_computed_task(
         self,
-        timestamp                   = None,
-        ack_report_computed_task    = None,
-        requestor_private_key       = None,
-    ):
+        timestamp: Union[str, datetime.datetime, None] = None,
+        ack_report_computed_task: Optional[AckReportComputedTask] = None,
+        requestor_private_key: Optional[bytes] = None,
+    ) -> bytes:
         with freeze_time(timestamp or self._get_timestamp_string()):
             return dump(
                 ack_report_computed_task,
                 requestor_private_key if requestor_private_key is not None else self.REQUESTOR_PRIVATE_KEY,
                 settings.CONCENT_PUBLIC_KEY
             )
 
-    def _parse_iso_date_to_timestamp(self, date_string):    # pylint: disable=no-self-use
+    def _parse_iso_date_to_timestamp(self, date_string) -> int:    # pylint: disable=no-self-use
         return int(dateutil.parser.parse(date_string).timestamp())
 
-    def _get_timestamp_string(self):                        # pylint: disable=no-self-use
+    def _get_timestamp_string(self) -> str:                        # pylint: disable=no-self-use
```

Comment:
Ok, if it has to stay just delete spaces between `str:` and pylint disable

Reply:
it will be function now and pylint disable will be deleted :)